pkg/rtc/mediatrack.go

```diff
@@ -221,6 +238,12 @@ func (t *MediaTrack) AddReceiver(receiver *webrtc.RTPReceiver, track *webrtc.Tra
 	})
 }
 
+func (t *MediaTrack) TrySetSimulcastSSRC(layer uint8, ssrc uint32) {
```

Comment:
@cnderrauber Saw some discussion between you and @davidzhao on lock for track info. I saw that `layerSsrcs` did not seem to have a lock before also and so does `ToProto`. So, just keeping them without locks and moving it into `MediaTrack` as `RemoteMediaTrack` does not have this.

Reply:
Thanks!